cmake/FindEclipseClp.cmake

```diff
@@ -25,7 +28,7 @@ if (ECLIPSE_BIN)
 			set(ECLIPSE_CFLAGS -I${ECLIPSE_INCDIR})
 			set(ECLIPSE_LDFLAGS "-L${ECLIPSE_LIBDIR} -Wl,-R${ECLIPSE_LIBDIR} -leclipse")
 			set(ECLIPSE_LIBRARY_DIRS ${ECLIPSE_LIBDIR})
-			set(ECLIPSE_INCLUDE_DIRS ${ECLIPSE_INCDIR})
+			set(ECLIPSE_INCLUDE_DIRS /usr/include/eclipse-clp ${ECLIPSE_INCDIR})
```

Comment:
Wait, why is there an include path hardcoded here?

Reply:
Because eclipse-clp does not provide a pkg-config file and `ECLIPSE_INCDIR` is also in some sense hard-coded, as it doesn't check whether the directory actually exists and just assumes it will be `${ECLIPSE_DIR}/include/${ECLIPSE_HOSTARCH}$`.

Of course the best thing would be to use `#include <eclipse-clp/eclipseclass.h>`, but that would break compatibility with the default installation layout of ECLiPSe.

Reply:
Fixed in 99504677a8df72e1509114e552b933137430852a